### PR TITLE
add alertmanager alerts

### PR DESCRIPTION
**Pull request by pb82**
add alerts for alertmanager

### CODE REVIEW

**Comment by KevFan** [comment]
Alerts added to alertmanager :+1: 
Screenshot from 2020-08-24 16-30-40

Though I'm not sure are the alertmanager metrics scraped as I couldn't find the `alertmanager_notifications_failed_total` and `alertmanager_config_last_reload_successful` metrics available in prometheus

Doesn't look like there is a service monitor for alertmanager in the namespace :thinking: 
Screenshot from 2020-08-24 17-21-32

**Comment by pb82** [comment]
@david-martin do the alerts look ok to you?

**Comment by pb82** [comment]
@KevFan with AMO 0.0.30, the alerts should be imported through the existing prometheus service monitor.

File: roles/middleware_monitoring_config/templates/alertmanager_alerts.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    - name: alertmanager.rules
+      rules:
+        - alert: AlertmanagerFailedNotifications
+          annotations:
+            message: Alertmanager has failed to deliver notifications
+          expr: |
+            rate(alertmanager_notifications_failed_total[1m]) > 0
+          for: 1m
+          labels:
+            severity: critical
```
Comment (david-martin) [inline roles/middleware_monitoring_config/templates/alertmanager_alerts.yml.j2:18]: Given this is critical, it needs to pass the sre critical criteria
(apologies for private link https://mojo.redhat.com/docs/DOC-1230703)

**Comment by KevFan** [comment]
Verified :+1: 

Screenshot from 2020-08-25 15-43-04
Screenshot from 2020-08-25 15-45-01
Screenshot from 2020-08-25 16-03-58
Screenshot from 2020-08-25 16-17-18

**Pull request merged**